docs/symphony-book/concepts/unified-object-model/campaign.md

```diff
@@ -37,7 +37,7 @@ type IStageProvider interface {
 
 Essentially, a stage provider takes the inputs, performs any actions, and returns the outputs. When a stage is invoked, activation inputs are provided through the `inputs` parameter, plus any input parameter declared on the stage itself. For example, if a campaign is activated with inputs `foo` and `bar`, and the stage definition contains an input `baz`, the `inputs` parameter will contain all the three values. 
 
-> **NOTE**: In current version, inputs defined in stage definition override activation inputs. If you do want to keep the activation input, you can use expression `$input(<field name>)` in your stage definition to carry over activation input.
+> **NOTE**: In current version, inputs defined in stage definition. If you do want to get value from the activation input, you can use expression `$trigger(<field name>, <default value>)` in your stage definition to carry over activation input.
```

Comment:
can we add an example of how to use this expression in the doc

Reply:
I think so. After checking with @lirenjie95 , seems in existing docs, we don't have any samples to show how to use input value from activation. We need to add this.

Reply:
Added in the new version.